tests/nuodb_basic_test.py

```diff
@@ -172,6 +172,23 @@ def test_many_significant_digits(self):
     def test_numeric_no_decimal(self):
         self._test_decimal_fixture(decimal.Decimal("1.000"), 5, 3)
 
+    # This test case produces at least three different defects, perhaps
+    # more under the covers.
+    @unittest.skip("produces data errors for e-notations")
+    def test_enotation_decimal_large(self):
+        numbers = (
+            # Yields:  AssertionError: '400000000.00' != '4E+8'
+            decimal.Decimal('4E+8'),
```

Comment:
So this is not a departure from the prior approach and fixes contributed; if you look at the implementation of getScaledInt that I wrote it uses this sort of syntax:

decimal.Decimal((0,(4,),8)),

Again very common for database integrators support of decimals. The value for the above as it turns out is 4E+8. I think if you look into it the approach works wonderfully.